code/modules/clothing/suits/misc_suits.dm

```diff
@@ -1196,3 +1196,87 @@
 	body_parts_covered = UPPER_TORSO|ARMS
 	cold_protection = UPPER_TORSO | ARMS
 	min_cold_protection_temperature = FIRE_SUIT_MIN_TEMP_PROTECT
+
+/obj/item/clothing/suit/hooded/abaya
+	name = "abaya"
+	desc = "A modest, unrevealing attire fitted with a veil."
+	icon_state = "abaya"
+	item_state = "abaya"
+	body_parts_covered = UPPER_TORSO|LOWER_TORSO|LEGS|ARMS
+	allowed = list(/obj/item/storage/bible, /obj/item/nullrod, /obj/item/reagent_containers/food/drinks/bottle/holywater, /obj/item/storage/fancy/candle_box, /obj/item/candle, /obj/item/tank/internals/emergency_oxygen)
+	hoodtype = /obj/item/clothing/head/hooded/screened_niqab
+	flags_inv = HIDEJUMPSUIT
+	var/list/options = list(
+		"Abaya" = /obj/item/clothing/suit/hooded/abaya,
+		"Red Abaya" = /obj/item/clothing/suit/hooded/abaya/red,
+		"Orange Abaya" = /obj/item/clothing/suit/hooded/abaya/orange,
+		"Yellow Abaya" = /obj/item/clothing/suit/hooded/abaya/yellow,
+		"Green Abaya" = /obj/item/clothing/suit/hooded/abaya/green,
+		"Blue Abaya" = /obj/item/clothing/suit/hooded/abaya/blue,
+		"Purple Abaya" = /obj/item/clothing/suit/hooded/abaya/purple,
+		"White Abaya" = /obj/item/clothing/suit/hooded/abaya/white,
+		"Rainbow Abaya" = /obj/item/clothing/suit/hooded/abaya/rainbow
+	)
+
+	sprite_sheets = list(
+	"Vox" = 'icons/mob/clothing/species/vox/suit.dmi',
+	"Grey" = 'icons/mob/clothing/species/grey/suit.dmi',
+	"Drask" = 'icons/mob/clothing/species/drask/suit.dmi',
+	"Kidan" = 'icons/mob/clothing/species/kidan/suit.dmi'
+	)
+
+/obj/item/clothing/suit/hooded/abaya/proc/reskin_abaya(mob/living/L)
+	var/choice = input(L,"You may only change the color once.","Reskin Abaya") in options
+
+	if(!options[choice] || HAS_TRAIT(L, TRAIT_HANDS_BLOCKED) || !in_range(L, src))
+		return
+	var/abaya_type = options[choice]
+	var/obj/item/clothing/suit/hooded/abaya/abaya = new abaya_type(get_turf(src))
+	L.unEquip(src, silent = TRUE)
+	L.put_in_active_hand(abaya)
+	to_chat(L, "You are now wearing a [choice]. Allahu Akbar!")
```

Comment:
Peace and blessings be upon you. It says, outdated?

Reply:
That's because you changed this line of code earlier, meaning that the suggestion isn't completely up to date. You can still add the `/` in VSC and commit it again, and then you can resolve this conversation :)